stage1/stop/stop.go

```diff
@@ -32,7 +33,7 @@ func init() {
 	flag.BoolVar(&force, "force", false, "Forced stopping")
 }
 
-func readIntFromFile(path string) (i int, err error) {
+func readIntFromFile(path string) (i int32, err error) {
```

Comment:
Why the change from "int" to "int32"?

We now have 4 functions `readIntFromFile` and half of them use a `int`, the other half uses a `int32`:
```
$ git grep -n  'readIntFromFile(path'
rkt/pods.go:716:func (p *pod) readIntFromFile(path string) (i int, err error) {
stage1/stop/stop.go:36:func readIntFromFile(path string) (i int32, err error) {
stage1/stop_kvm/stop_kvm.go:39:func readIntFromFile(path string) (i int, err error) {
stage1_fly/stop/stop.go:36:func readIntFromFile(path string) (i int32, err error) {
```